settings.yml

```diff
@@ -12,8 +12,8 @@ variables:
   ########################
 
   # the 'localToken_' prefix will be removed from the key name when the pipelines run.
-  # e.g. if you have a token in your parameter file as [[customKey]], then the token defined in this file looks like "localToken_customKey": 'value'
-  localToken_namePrefix: '' # A 3-5 character length unique string, included in the resources names (e.g. 'cntso'). Used for local module testing and pipelines.
+  # e.g. if you have a token in your parameter file as <<customKey>>, then the token defined in this file looks like "localToken_customKey": 'value'
+  localToken_namePrefix: 'lsno' # A 3-5 character length unique string, included in the resources names (e.g. 'cntso'). Used for local module testing and pipelines.
```

Comment:
Another leftover 😉

Reply:
I guess it makes sense to undo all changes in this file

Reply:
Yes, needed my local settings.yml for pipeline run in my branch. Reverted to carml settings now that pipeline is green.